binary_search/binary_search.py

```diff
@@ -0,0 +1,54 @@
+from random import randint
+
+
+def binary_search_iterative(array, left, right, element):
+    while left <= right:
+        mid = int(left + (right - left) / 2)
+        # indices of a list must be integer
+        if array[mid] == element:
+            return mid
+        elif array[mid] < element:
+            left = mid + 1
+        else:
+            right = mid - 1
+    return -1
+
+
+def binary_search_recursive(array, left, right, element):
```

Comment:
@singhpratyush if we set left =0 in each recursion would that give correct answer?
suppose our element is in the right half then we set left = mid+1, wouldnt that create a problem?